mapie/classification.py

```diff
@@ -250,18 +252,18 @@ def _check_depreciated(self) -> None:
         if self.method == "score":
             warnings.warn(
                 "WARNING: Deprecated method. "
-                + "The method \"score\" is outdated. "
-                + "Prefer to use \"lac\" instead to keep "
+                + 'The method "score" is outdated. '
+                + 'Prefer to use "lac" instead to keep '
                 + "the same behavior in the next release.",
-                DeprecationWarning
+                DeprecationWarning,
             )
         if self.method == "cumulated_score":
             warnings.warn(
                 "WARNING: Deprecated method. "
-                + "The method \"cumulated_score\" is outdated. "
-                + "Prefer to use \"aps\" instead to keep "
+                + 'The method "cumulated_score" is outdated. '
```

Comment:
Are we sure we wanted those backslash removed ?

Reply:
The quotation marks " and ' have been interchanged. So I think there's no problem.